chore: add a pull request template (#202)
tact-lang/tact: .github/workflows/pull_request_template.md

Closes #number

<!--
IMPORTANT:
If your PR doesn't close a particular issue, please, create the issue first and describe the whole context: what you're adding/changing and why you're doing so. And only then open the Pull Request, which would close that issue!
-->

* [ ] I have updated CHANGELOG.md

> * [ ] I have added unit tests to check possible regressions (if your PR introduces new error messages, each of those should have a distinct test case)

Author: ```suggestion
* [ ] I have added unit tests to check possible regressions (if your PR introduces new error messages, each of those should have a distinct test case)
* [ ] I have run all the tests locally and no test failure was reported
```

Closes #number

<!--
IMPORTANT:
If your PR doesn't close a particular issue, please, create the issue first and describe the whole context: what you're adding/changing and why you're doing so. And only then open the Pull Request, which would close that issue!
-->

* [ ] I have updated CHANGELOG.md

> * [ ] I have added unit tests to check possible regressions (if your PR introduces new error messages, each of those should have a distinct test case)

Author: ```suggestion
* [ ] I have added unit tests to demonstrate the contribution is correctly implemented
```

> Closes issue #number

Reviewer: ```suggestion
Closes #number

<!--
IMPORTANT:
If your PR doesn't close a particular issue, please, create the issue first and describe the whole context: what you're adding/changing and why you're doing so. And only then open the Pull Request, which would close that issue!
-->
```

Author: > If your PR doesn't close a particular issue, please, add the context block instead:

yeah, this is what we are trying to avoid in general: PRs without corresponding issues

Reviewer: Agreed, changed the hidden note, so that it asks to create issues with context first, and then submit a PR.
